build.gradle.kts

```diff
@@ -148,14 +147,14 @@ class MultilineFormatter : Serializable {
 
 tasks.withType<JavaCompile>().configureEach {
     options.compilerArgs.addAll(listOf("-Xlint:all", "-Werror"))
-    if (name == "compileJava") {
-        options.errorprone {
-            disableWarningsInGeneratedCode.set(true)
-            option("NullAway:AnnotatedPackages", "com.mongodb.hibernate")
-            error("NullAway")
-        }
-    } else {
-        options.errorprone.isEnabled.set(false)
+    when (this) {
+        tasks.compileJava.get() ->
+            options.errorprone {
+                disableWarningsInGeneratedCode = true
+                option("NullAway:AnnotatedPackages", "com.mongodb.hibernate")
+                error("NullAway")
+            }
+        else -> options.errorprone.isEnabled = false
```

Comment:
recently I am learning Kotlin language. The above changes are to make our Kotlin usage more idiomatic.